rocminfo/PKGBUILD

```diff
@@ -7,7 +7,7 @@ pkgdesc="ROCm info tools - rocm_agent_enumerator"
 arch=('x86_64')
```

Comment:
Should `pkgrel` be bumped up here as well?

Reply:
Ugh, yes, in every package changed, but there are so many that a couple slippepd

Reply:
Fixed!